src/utils/meta.tsx

```diff
@@ -10,7 +10,7 @@ interface Meta {
 }
 
 const getFullSlugFromStory = (story?: Story) =>
-  story && story.full_slug.replace(/\/?home$/, '')
+  story && story.full_slug.replace(/\/?home$/, '').replace(/\/sv($|\/)/, '')
```

Comment:
`/^\/sv($|\/)/`?

Reply:
I cant tell whether this is correct or not 🤔

Reply:
actually, as the `sv` part is [being sanitized](https://github.com/HedvigInsurance/web-next/pull/241/files#diff-32c50dfc6f7e52967a9eaa0e836e0d0cR109) i think this might not even be needed